translate/convert/xliff2po.py

```diff
@@ -67,11 +69,6 @@ def converttransunit(self, transunit):
 
     def convertstore(self, inputfile, duplicatestyle="msgctxt"):
         """Converts a .xliff file to .po format"""
-        # XXX: The inputfile is converted to string because Pootle supplies
-        # XXX: a PootleFile object as input which cannot be sent to PoXliffFile
-        # XXX: The better way would be to have a consistent conversion API.
-        if not isinstance(inputfile, (file, wStringIO.StringIO)):
-            inputfile = str(inputfile)
```

Comment:
That probably was used in old Pootle and now is gone. Please craft the commit I asked for in https://github.com/translate/translate/pull/3339/files#r39437681